### PR TITLE
Content loading issue suraj

### PR DESCRIPTION
**Pull request by Dhan-shri**
This PR is raised for the replacing the pathway api to new pathways/dropdown API to reduce the loading time

### CODE REVIEW

**Review by surajsahani — APPROVED** [review]
LGTM

**Comment by Dhan-shri** [comment]
@damanpreet97 can you also please check this..

File: learn/src/main/java/org/navgurukul/learn/courses/db/CoursesDatabase.kt [hunk 1]
```diff
@@ -371,6 +371,15 @@ val MIGRATION_12_13 = object : Migration(12, 13){
 
 }
 
+val MIGRATION_13_14 = object : Migration(13, 14){
```
Comment (damanpreet97) [inline learn/src/main/java/org/navgurukul/learn/courses/db/CoursesDatabase.kt:374]: Is this migration is just for dropping shouldShowCertificate column?

Comment (Dhan-shri) [reply learn/src/main/java/org/navgurukul/learn/courses/db/CoursesDatabase.kt:374]: Yes

Comment (Dhan-shri) [reply learn/src/main/java/org/navgurukul/learn/courses/db/CoursesDatabase.kt:374]: We are using same table for new api as well, as almost structure is same for both

**Pull request merged**